ioctl/client.go

```diff
@@ -235,13 +239,28 @@ func (c *client) AliasMap() map[string]string {
 	return aliases
 }
 
-func (c *client) WriteConfig(cfg config.Config) error {
-	out, err := yaml.Marshal(&cfg)
+func (c *client) SetAlias(alias string, addr string) error {
+	aliases := c.AliasMap()
+	for aliases[addr] != "" {
```

Comment:
Don't use `for {}` here

Reply:
see  https://github.com/iotexproject/iotex-core/blob/182c040a6c7a7922965ab4679b072f29ccd941b1/ioctl/cmd/alias/aliasset.go#L56

Reply:
> see
> 
> https://github.com/iotexproject/iotex-core/blob/182c040a6c7a7922965ab4679b072f29ccd941b1/ioctl/cmd/alias/aliasset.go#L56

You could improve it in this pr

Reply:
Could you help fix https://github.com/iotexproject/iotex-core/issues/3171 when free?

Reply:
ok